hedera-node/docs/design/services/smart-contract-service/frictionless-airdrops-system-contracts.md

```diff
@@ -0,0 +1,165 @@
+| `0xc1acfe5d`           | `function cancelAirdrops(PendingAirdrop[] memory pendingAirdrops) external returns (int64 responseCode)`                            | `ResponseCode`                           | The response code from the call                                           |
+| `0x241b6af7`           | `function claimAirdrops(PendingAirdrop[] memory pendingAirdrops) external returns (int64 responseCode)`                             | `ResponseCode`                           | The response code from the call                                           |
+| `0x012ea0b1`           | `function rejectTokens(address[] memory ftAddresses,  NftId[] memory nftIds) external returns (int64 responseCode)`                 | `ResponseCode`                           | The response code from the call                                           |
+| `0xc60ffe1b`           | `function getPendingAirdrops(address account) external returns (int64 responseCode, PendingAirdropRecord[] memory pendingAirdrops)` | `ResponseCode`, `PendingAirdropRecord[]` | The response code from the call and the pending airdrops for given sender |
+
+New system contract functions must be added to the `IHRC` interface to support airdropping tokens.
+
+| Function Selector Hash |                                                Function Signature                                                |    Response    |                                 |
+|------------------------|------------------------------------------------------------------------------------------------------------------|----------------|---------------------------------|
+| `0xcef5b705`           | `function cancelAirdropFT(address receiverAddress) external returns (uint256 responseCode)`                      | `ResponseCode` | The response code from the call |
+| `0xad4917cf`           | `function cancelAirdropNFT(address receiverAddress, int64 serialNumber) external returns (uint256 responseCode)` | `ResponseCode` | The response code from the call |
+| `0xa83bc5b2`           | `function claimAirdropFT(address senderAddress) external returns (uint256 responseCode)`                         | `ResponseCode` | The response code from the call |
+| `0x63ada5d7`           | `function claimAirdropNFT(address senderAddress, int64 serialNumber) external returns (uint256 responseCode)`    | `ResponseCode` | The response code from the call |
+| `0x76c6b391`           | `function rejectTokenFT() external returns (uint256 responseCode)`                                               | `ResponseCode` | The response code from the call |
+| `0xedd9c546`           | `function rejectTokensNFT(int64[] memory serialNumber) external returns (uint256 responseCode)`                  | `ResponseCode` | The response code from the call |
```

Comment:
Q: Should this be as follows?
```suggestion
| `0xedd9c546`           | `function rejectTokenNFTs(int64[] memory serialNumbers) external returns (uint256 responseCode)`                  | `ResponseCode` | The response code from the call |
```

Reply:
Yes, Thanks! This was the way it was described in the HIP, but it definitely reads better this way.

Reply:
Okay, we should correct it, I seem to have missed it so the above is a correction on my own work :)